app/Livewire/Auth/Login.php

```diff
@@ -43,7 +43,7 @@ public function login(): void
         RateLimiter::clear($this->throttleKey());
         Session::regenerate();
 
-        $this->redirectIntended(default: route('dashboard', absolute: false), navigate: true);
+        $this->redirectIntended(default: route('videos', absolute: false), navigate: true);
```

Comment:
Correct route should be 'videos.index' as shown in routes/web.php
```suggestion
        $this->redirectIntended(default: route('videos.index', absolute: false), navigate: true);
```